ci/docker/public_key.asc

```diff
@@ -0,0 +1,41 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Whose public key is this? It's a spack binary cache key? Might it be a good idea to add something more descriptive in the name?

Reply:
Yeah, I think that's already better! Especially as in the SIRIUS case the key is in a `spack` subdirectory and here it's in `docker` so doesn't even have `spack` in the path.